fc/fullcontact.go

```diff
@@ -44,7 +68,15 @@ func (fcClient *fullContactClient) autoRetry(ch chan *APIResponse, err error, re
 	if retryAttemptsDone < min(fcClient.retryHandler.RetryAttempts(), 5) {
 		retryAttemptsDone++
 		time.Sleep(time.Duration(fcClient.retryHandler.RetryDelayMillis()*(1<<(retryAttemptsDone-1))) * time.Millisecond)
-		req, err := fcClient.newHttpRequest(url, reqBytes)
+		var req *http.Request
+		var err error
+		if url == emailVerificationUrl {
```

Comment:
this is in at least 2 spots -- is there a way to make a list of GET urls, check whether the URL is in this list, and otherwise it's a POST? I'm thinking for when we add a third GET, we don't need to add it in two spots? Just a thought.

Reply:
Yeah, good point, updated